### ISSUE TITLE
Add term: complex XY

### ISSUE THREAD

**Issue by enemeshazi** [Tree-of-Sex/tos-ontology#45]
### In which category is your new term request?

- [X] new_definition
- [X] existing_definition

### Preferred term label

complex XY

### Synonyms

complex male-heterogametic system

### Textual definition

A male-heterogametic system of gonochoristic organisms where sex is determined by a set of multiple sex chromosomes instead of just a sinlge sex-chromosome pair. Individuals carrying the chromosome(s) occurring only in one sex (Y chromosome) develop male gonads.

### Attribution

0000-0002-0179-1153

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ASEMBER2** [comment]
I do not agree with this explanation since it does not provide a clear distinction between multiple sex chromosome systems (which have some specific definition) and complex sex chromosome systems. Please, see my defined term "multiple sex chromosomes" where I touch this issue. Particularly I copy the the relevant part here:

Exclude polygenic or polyfactorial sex determination (this is something different). Multiple sex chromosomes are usually monogenic but contain more chromosomes due to structural rearrangements (or as a consequence of aneuploidy). In case where "polygenic" means two sex chromosome systems in the transient state of sex chromosome turnover (i.e. previous sex chromosome system is still co-segregating with a new dominant sex chromosome system, such as e.g. in cichlids, e.g. Metriaclima pyrsonotus) I rather suggest to call it "complex sex chromosome system". The same may apply if there are populations with different sex chromosome systems and they mate and create different combinations of sex chromosome systems (such as e.g. in Glandirana rugosa, Xiphophorus maculatus, Xenopus tropicalis).

**Comment by ValenzuelaLab** [comment]
I suggest the following definition:

A male-heterogametic system in gonochoristic organisms where sex is determined by multiple sex chromosomes instead of a single XX/XY sex-chromosome pair, where individuals carrying the Y chromosome(s) develop as males (e.g. X1X2Y, XY1Y2, or X1Y1X2Y2X3Y3X4Y4X5Y5).

**Comment by ASEMBER2** [comment]
I agree and in that context we should not forget on complex X0 (such as in spiders with multiple different Xs but lacking Y, or in some orthopterans etc.).

**Comment by enemeshazi** [comment]
> I suggest the following definition:
> 
> A male-heterogametic system in gonochoristic organisms where sex is determined by multiple sex chromosomes instead of a single XX/XY sex-chromosome pair, where individuals carrying the Y chromosome(s) develop as males (e.g. X1X2Y, XY1Y2, or X1Y1X2Y2X3Y3X4Y4X5Y5).

Approved

**Comment by ValenzuelaLab** [comment]
Ah! Beautiful spiders! Should we try to accommodate them here under complex XY or instead create a term for Complex X0?

If so, we could also have a separate term for Complex Male-Heterogametic Systems that encompasses complex XY and complex X0 sex determination.

**Comment by genomicrocosm** [comment]
What about ZW systems, do we need to create a term for "complex ZW"?

**Comment by genomicrocosm** [comment]
The term "complex" might be a bit misleading, how about "multiple" instead?

**Comment by guigueny-crypto** [comment]
agree that "complex" is misleading.
